embedded/sql/parser.go

```diff
@@ -34,8 +34,9 @@ var reservedWords = map[string]int{
 	"DATABASE":       DATABASE,
 	"SNAPSHOT":       SNAPSHOT,
 	"SINCE":          SINCE,
-	"UP":             UP,
```

Comment:
Wouldn't this be backwards incompatible?

Reply:
no, the UP token was not used...